src/index.html

```diff
@@ -1,15 +1,144 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+     autocomplete="off"
+     >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field" for="surnmae">
+          Surname:
+          <input type="text" id="surnmae"
+            name="surnmae" required>
+        </label>
```

Comment:
3 or more than 3 attributes -> 1 attribute per 1 line
fix everywhere

```suggestion
        <label class="form-field" for="surnmae">
          Surname:
          <input
            type="text"
            id="surname"
            name="surname"
            required
          >
        </label>
```